sdk/textanalytics/Azure.AI.TextAnalytics/samples/Sample_AnalyzeBatchActions.md

```diff
@@ -68,88 +60,117 @@ To run multiple actions in multiple documents, call `StartAnalyzeBatchActionsAsy
             }
-            Console.WriteLine("");
         }
 
         Console.WriteLine("Recognized Linked Entities");
-
-        foreach (RecognizeLinkedEntitiesResult result in linkedEntitiesResult)
+        docNumber = 1;
+        foreach (RecognizeLinkedEntitiesActionResult linkedEntitiesActionResults in entityLinkingActionsResults)
         {
-            Console.WriteLine($"  Recognized the following {result.Entities.Count} linked entities:");
-
-            foreach (LinkedEntity entity in result.Entities)
+            foreach (RecognizeLinkedEntitiesResult result in linkedEntitiesActionResults.Result)
             {
-                Console.WriteLine($"  Entity: {entity.Name}");
-                Console.WriteLine($"  DataSource: {entity.DataSource}");
-                Console.WriteLine($"  DataSource EntityId: {entity.DataSourceEntityId}");
-                Console.WriteLine($"  Language: {entity.Language}");
-                Console.WriteLine($"  DataSource Url: {entity.Url}");
-
-                Console.WriteLine($"  Total Matches: {entity.Matches.Count()}");
-                foreach (LinkedEntityMatch match in entity.Matches)
+                Console.WriteLine($" Document #{docNumber++}");
+                Console.WriteLine($"  Recognized the following {result.Entities.Count} linked entities:");
+
+                foreach (LinkedEntity entity in result.Entities)
                 {
-                    Console.WriteLine($"    Match Text: {match.Text}");
-                    Console.WriteLine($"    ConfidenceScore: {match.ConfidenceScore}");
-                    Console.WriteLine($"    Offset: {match.Offset}");
-                    Console.WriteLine($"    Length: {match.Length}");
+                    Console.WriteLine($"  Entity: {entity.Name}");
+                    Console.WriteLine($"  DataSource: {entity.DataSource}");
+                    Console.WriteLine($"  DataSource EntityId: {entity.DataSourceEntityId}");
+                    Console.WriteLine($"  Language: {entity.Language}");
+                    Console.WriteLine($"  DataSource Url: {entity.Url}");
+
+                    Console.WriteLine($"  Total Matches: {entity.Matches.Count()}");
+                    foreach (LinkedEntityMatch match in entity.Matches)
+                    {
+                        Console.WriteLine($"    Match Text: {match.Text}");
+                        Console.WriteLine($"    ConfidenceScore: {match.ConfidenceScore}");
+                        Console.WriteLine($"    Offset: {match.Offset}");
+                        Console.WriteLine($"    Length: {match.Length}");
+                    }
+                    Console.WriteLine("");
                 }
                 Console.WriteLine("");
             }
-            Console.WriteLine("");
+        }
+
+        Console.WriteLine("Analyze Sentiment");
+        docNumber = 1;
+        foreach (AnalyzeSentimentActionResult analyzeSentimentActionsResult in analyzeSentimentActionsResults)
```

Comment:
hmmm, I thought you were against bloating the sample with sentiment analysis before.

Reply:
Not against. I was suggesting that mined opinion output didn't add much to the sample that JS had and it made it lengthy.
In this case, I am showing in a simple way how the analyze sentiment results are back.